tests/models/rwkv5/test_tokenization_rwkv5.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers.testing_utils import require_torch
+from transformers.utils import is_torch_available
+
+
+if is_torch_available():
+    from transformers import RWKVWorldTokenizer
+
+
+@require_torch
+class RWKVWorldTokenizationTest(unittest.TestCase):
```

Comment:
let's make sure the tokenizer is fully tested with the TokenizerTesterMixin

Reply:
Because the RWKVWorldHFTokenizer has some special settings, such as eos_id=bos_id=0, many tests fail when using the TokenizerTesterMixin to test the tokenizer. Therefore, can we not use the TokenizerTesterMixin for testing in this case?

Reply:
No I think the way to fix this is to properly set eos_token and bos_token no?